test/testinput/bufr_ncep_satwind_avhrr.yaml

```diff
@@ -0,0 +1,169 @@
+      obsdatain: "./testinput/bufr_ncep_satwind_avhrr.bufr_d"
+
+      exports:
+        variables:
+          satelliteIdentifier:
+            query: "*/SAID"
+
+          latitude:
+            query: "*/CLAT"
+
+          longitude:
+            query: "*/CLON"
+
+          timestamp:
+            datetime:
+              year: "*/YEAR"
+              month: "*/MNTH"
+              day: "*/DAYS"
+              hour: "*/HOUR"
+              minute: "*/MINU"
+              second: "*/SECO"
+
+          satelliteDerivedWindComputationMethod:
+            query: "*/SWCM"
+
+          satelliteZenithAngle:
+            query: "*/SAZA"
+
+          originatingCenter:
+            query: "*/OGCE"
+
+          satelliteChannelCentreFrequency:
+            query: "*/SCCF"
+
+          sdmeditSatWindQualMark:
+            query: "*/SWQM"
+
+          heightAssignmentMethod:
+            query: "*/HAMD"
+
+          pressure:
+            query: "*/PRLC"
+
+          windDirection:
+            query: "*/WDIR"
+
+          windSpeed:
+            query: "*/WSPD"
+
+          windSpeedGeneratingApplication:
+            query: "*/GQCPRMS/GNAP[1]"
+
+          windDirectionGeneratingApplication:
+            query: "*/GQCPRMS/GNAP[2]"
+
+          windSpeedPercentConfidence:
+            query: "*/GQCPRMS/PCCF[1]"
+
+          windDirectionPercentConfidence:
+            query: "*/GQCPRMS/PCCF[2]"
+
+    ioda:
+      backend: netcdf
+      obsdataout: "./testoutput/bufr_ncep_satwind_avhrr.nc"
+      dimensions:
+        - name: Confidence
+          paths:
+            - "*/GQCPRMS"
+
+      variables:
+        - name: "MetaData/satelliteIdentifier"
+          source: variables/satelliteIdentifier
+          longName: "Satellite Identifier"
+          units: ""
+
+        - name: "MetaData/latitude"
+          source: variables/latitude
+          longName: "Latitude (Coarse Accuracy)"
+          units: "degree_north"
+
+        - name: "MetaData/longitude"
+          source: variables/longitude
+          longName: "Longitude (Coarse Accuracy)"
+          units: "degree_east"
+
+        - name: "MetaData/dateTime"
+          coordinates: "longitude latitude"
+          source: variables/timestamp
+          longName: "Datetime"
+          units: "seconds since 1970-01-01T00:00:00Z"
+
+        - name: "MetaData/satelliteDerivedWindComputationMethod"
+          source: variables/satelliteDerivedWindComputationMethod
+          longName: "Satellite Derived Wind Computation Method"
+          units: ""
+
+        - name: "MetaData/satelliteZenithAngle"
+          source: variables/satelliteZenithAngle
+          longName: "Satellite Zenith Angle"
+          units: "degree"
+
+        - name: "MetaData/originatingCenter"
+          source: variables/originatingCenter
+          longName: "Identification of Originating/Generating Center"
+          units: ""
+
+        - name: "MetaData/satelliteChannelCentreFrequency"
```

Comment:
Change to MetaData/sensorCentralFrequency

Reply:
The yaml has been updated.
```
          sensorCentralFrequency:
            query: "*/SCCF"

        - name: "MetaData/sensorCentralFrequency"
          source: variables/sensorCentralFrequency
          longName: "Satellite Channel Center Frequency"
          units: "Hz"
```